lib/actions-oclif/app/index.ts

```diff
@@ -57,10 +57,17 @@ export default class AppCmd extends Command {
 	public async run() {
 		const { args: params } = this.parse<FlagsDef, ArgsDef>(AppCmd);
 
-		const application = await getBalenaSdk().models.application.get(
+		const application = (await getBalenaSdk().models.application.get(
 			tryAsInteger(params.name),
-		);
+			{
+				$expand: {
+					is_for__device_type: { $select: 'slug' },
+				},
+			},
+		)) as ApplicationWithDeviceType;
 
+		// @ts-expect-error
+		application.device_type = application.is_for__device_type[0].slug;
```

Comment:
I think something went wrong around here as well, as the output is missing the commit line:
```
$ ./bin/balena app test-rpi
== TEST RPI
ID:          1301645
DEVICE TYPE: raspberrypi3
SLUG:        gh_paulo_castro/test-rpi
```
With v12.10.2:
```
./balena-cli-12.10.2/balena app test-rpi
== TEST RPI
ID:          1301645
DEVICE TYPE: raspberrypi3
SLUG:        gh_paulo_castro/test-rpi
COMMIT:      8ebfde590a2794b00486633eeff32812
```